### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by chris1284**
Updated widget

### CODE REVIEW

File: www/tablet/js/widget_calview.js [hunk 1]
```diff
@@ -25,14 +26,14 @@ var Modul_calview = function () {
         me.elements = $('div[data-type="' + me.widgetname + '"]:not([data-ready])', me.area);
         me.elements.each(function (index) {
             var elem = $(this);
-            elem.attr("data-ready", "");
+	    elem.attr("data-ready", ""
```
Comment (knowthelist) [inline www/tablet/js/widget_calview.js]: Why this change to break correct syntax? By mistake?

**Comment by florian-asche** [comment]
Vielen Dank!
